db/migrate/20150813202625_set_missing_password_digest_in_authentications.rb

```diff
@@ -0,0 +1,19 @@
+class SetMissingPasswordDigestInAuthentications < ActiveRecord::Migration
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Updating password_digest") do
+      Authentication.where(:password_digest => nil).each do |auth|
+        auth.update_attributes(:password_digest => BCrypt::Password.create(auth.password))
+      end
+    end
+  end
+
+  def down
+    say_with_time("Resetting password_digest") do
+      Authentication.update_all(:password_digest => nil)
+    end
```

Comment:
I don't think this is needed...because the next down migration will just delete the column entirely.